tests/Organization1655.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2020 Alexandre Delisle <alexodelisle  @gmail.com>                  #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+from . import Framework
+
+
+class Organization1655(Framework.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.org = self.g.get_organization("ThoughtCamera")
+
+    def testGetCredentialsAuth(self):
+        credentials_list = self.org.get_credential_authorizations()
+        self.assertListEqual(credentials_list, [])
```

Comment:
You also need to test delete, but I'd also prefer we return something so we can verify the returned attributes.

Reply:
I will need to wait in that case, since the only Organization I have available at the moment,with SAML SSO, is private. I can not display the credentials list in the --record. I will ask GitHub sales for a trial for Entreprise Org hosted on GitHub.